### PR TITLE
LED strip rework

### PR DESCRIPTION
**Pull request by gaelj**
CF & BF catchup.
Requires https://github.com/iNavFlight/inav-configurator/pull/28

### CODE REVIEW

**Comment by digitalentity** [comment]
@gaelj cool, thanks!

File: docs/LedStrip.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # LED Strip
 
-INAV supports the use of addressable LED strips.  Addressable LED strips allow each LED in the strip to
+Cleanflight supports the use of addressable LED strips.  Addressable LED strips allow each LED in the strip to
```
Comment (skaman82) [inline docs/LedStrip.md]: This should say INAV

Comment (gaelj) [inline docs/LedStrip.md]: Done. Thanks

**Comment by gaelj** [comment]
@digitalentity FYI, this has been merged into BF and now CF as well.

**Comment by digitalentity** [comment]
@martinbudden this involved miminal changes to dirvers, I believe merging this won't mess your effort of porting new IO, what do you think?

**Comment by martinbudden** [comment]
Yes, go ahead and merge it.

**Pull request merged**